test/TestUtils/RepoDirectoriesProvider.cs

```diff
@@ -28,28 +29,40 @@ public class RepoDirectoriesProvider
         public string NugetPackages => _nugetPackages;
         public string CorehostPackages => _corehostPackages;
         public string CorehostDummyPackages => _corehostDummyPackages;
+        public string DotnetSDK => _dotnetSDK;
 
         public RepoDirectoriesProvider(
             string repoRoot = null,
             string artifacts = null,
             string builtDotnet = null,
             string nugetPackages = null,
             string corehostPackages = null,
-            string corehostDummyPackages = null)
+            string corehostDummyPackages = null,
+            string dotnetSdk = null)
         {
-            _repoRoot = repoRoot ?? Path.Combine(Directory.GetCurrentDirectory(), "..", "..");
-            
-            string baseArtifactsFolder = artifacts ?? Path.Combine(_repoRoot, "artifacts");
+            _repoRoot = repoRoot ?? Directory.GetParent((Directory.GetCurrentDirectory())).Parent.FullName;
+
+            string baseArtifactsFolder = artifacts ?? Path.Combine(_repoRoot, "bin");
+
+            _dotnetSDK = dotnetSdk ?? Path.Combine(_repoRoot, "Tools", "dotnetcli");
 
             _targetRID = Environment.GetEnvironmentVariable("TEST_TARGETRID");
 
-            _artifacts = Path.Combine(baseArtifactsFolder, _targetRID);
+            _artifacts = Path.Combine(baseArtifactsFolder, _targetRID+".Debug");
+            if(!Directory.Exists(_artifacts))
```

Comment:
I'm not sure I like this pattern.  If I build for "Debug" on my machine, the Debug folder is going to exist.  If I then want to build/test Release builds, I need to remember to do a "clean", or else all these places are going to pick up the wrong directory.